src/main.rs

```diff
@@ -83,7 +89,10 @@ async fn main() -> Result<(), Box<dyn std::error::Error>>
         {
             setup::setup(matches).await.expect("Couldn't setup project properly")
         },
-        Some(("run", matches)) => run::run(matches).await?,
+        Some(("run", matches)) =>
+        {
+            run::run(matches).await?;
+        },
```

Comment:
what happened here?

Reply:
Ups, I was testing out returning something from `run::run`, but I didn't do it. I'll change it back